pinot-controller/src/test/java/org/apache/pinot/controller/helix/ControllerTest.java

```diff
@@ -721,6 +721,11 @@ public String reloadOfflineTable(String tableName, boolean forceDownload)
     return getControllerRequestClient().reloadTable(tableName, TableType.OFFLINE, forceDownload);
   }
 
+  public String needReloadOfflineTable(String tableNameWithType)
```

Comment:
Rename it to `checkIfReloadIsNeeded`. Does it only apply to offline table?

Reply:
It applies to both realtime and offline tables, changing the name